content/en/docs/contributing/e2e.md

```diff
@@ -104,3 +104,19 @@ to the X.509 SAN extension.
 
 Each test specifies a used feature using `s.checkFeatures(feature)`, which is then checked against the issuer's
 `UnsupportedFeatures` list. Tests which use a feature unsupported by an issuer are skipped for that issuer.
+
+### Cloud Provider Tests
+
+The master branch of cert-manager can also be tested against different cloud providers. Currently, tests for [EKS](https://aws.amazon.com/eks/) are present which run as a periodic job once every two days. These can also be manually triggered by commenting `/test aws-tests-pre` on any PR. 
```

Comment:
Add a TODO / caveat note to explain that the tests do not install cert-manager from the branch-under-test and link to an issue for that.